test/Test/Data/Mutable/Array.hs

```diff
@@ -227,7 +230,21 @@ lenResizeSeedTest :: Int -> Int -> ArrayTester
 lenResizeSeedTest newSize val arr =
   compInts
     (move newSize)
-    (move Linear.$ getSnd Linear.$ Array.length (Array.resizeSeed newSize val arr))
+    (move Linear.$ getSnd Linear.$ Array.length (Array.resize newSize val arr))
+
+resizeRef :: Property
+resizeRef = property $ do
+  l <- forAll list
+  n <- forAll $ Gen.int (Range.linear 0 (length l * 2))
+  let x = 42
+      expected = take n $ l ++ repeat x
```

Comment:
Maybe `cycle l` would make a slightly better quality infinite list.

Reply:
That `repeat` is there to give that expression the same semantics as `Array.resize`; using `cycle l` there would fail the test case when the array is growing.

Reply:
Got it. Then, maybe you want to generate `x` randomly. Just in case.